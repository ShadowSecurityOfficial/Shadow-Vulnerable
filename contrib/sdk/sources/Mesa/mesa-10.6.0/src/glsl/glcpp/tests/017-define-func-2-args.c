version https://git-lfs.github.com/spec/v1
oid sha256:b99d4471f332d95e1a240cc58abb6d0b6f9ec9c96b72e14f7c8450564b823fde
size 40
