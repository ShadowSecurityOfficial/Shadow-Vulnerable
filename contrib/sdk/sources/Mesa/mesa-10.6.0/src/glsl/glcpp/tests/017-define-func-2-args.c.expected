version https://git-lfs.github.com/spec/v1
oid sha256:8ecfa474e1323b2c94ccc8a297a14eb3058804d156c1e1b0af6ee2d06b440aa6
size 15
